tasks/us/census/acs.py

```diff
@@ -41,7 +41,7 @@ def tags(self):
             OBSTag(id='acs',
                    name='US American Community Survey',
                    type='source',
-                   description='The United States American Community Survey'),
+                   description='The United States American Community Survey, 2014 5-year sample. https://www.census.gov/programs-surveys/acs/'),
```

Comment:
Not necessarily -- we may also use 1 or 3 year data, so I would leave it as it was before.